eahub/templates/components/feedback.html

```diff
@@ -0,0 +1,79 @@
+{% block feedback %}
+{% load static %}
+  
+  <div id="overlay"></div>
+
+  <div id="feedback">
+  <!-- Button trigger modal -->
+  <button type="button" class="btn feedbackbtn" onClick="togglemodal()">
+      Feedback?
+    </button>
+    
+    <!-- Modal -->
+    <div class="modal fade" id="feedbackModal" tabindex="-1">
+      <div class="modal-content">
+          <div class="modal-header">
+            <h5 class="modal-title">Leave Feedback</h5>
+          </div>
+          <div class="modal-body">
+
+            <form id="form" class="form-group" method="post" action="/feedback/submit/">
+              <p>Do you have any feedback for us?</p>
+              {% csrf_token %}
+
+              <textarea class="form-control" name="message" cols="37" rows="5" required="required"></textarea>
+              <div id="errormsg" style="display:none" class='alert alert-danger'>
+                Feedback message field is required</div>
+            
+              <label for="email">Email (optional)</label>
+              <input class="form-control" type="text" name="email">
+              <input id="page_url" type="hidden" name="page_url" value="">
+              
+              <div class="modal-footer">
+                <button  id="closebtn" type="button" class="btn btn-secondary" onClick="togglemodal()">Close</button>
+                <input onClick="ajaxsubmit()" value="Submit" class="btn submitbtn" />
+              </div>
+              </form>
+
+              <div id="successmsg" style="display:none">
```

Comment:
![image](https://user-images.githubusercontent.com/15908208/106050771-79a8a100-60df-11eb-9a6f-32ceb44a18bd.png)

I can't seem to get out of this view once feedback has been submitted. We need a close button here

Reply:
Oh, I see, by clicking on the label. I don't think that's clear enough to the user. Would still add a close button

Reply:
When you're done and click on the label again, it still says thanks for your feedback. Is this on purpose? I'd say we'd rather reset in case someone wants to give more feedback.

@victor-yunenko, what do you think? I know you asked Lisa to prevent a refresh here. I assume you meant a refresh of the whole page, but would still prefer it if users could provide more feedback on clicking the label again?

Reply:
Finally. when I refresh the page and click on "feedback" again, I see the same feedback I've given before

Reply:
I like the idea, the form can have a simple success message then